src/GlobalDiffEq.jl

```diff
@@ -25,6 +27,71 @@ function DiffEqBase.__solve(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase
     return sol
 end
 
-export GlobalRichardson
+struct GlobalAdjoint{A} <: GlobalDiffEqAlgorithm
+    alg::A
+end
+
+function DiffEqBase.__solve(prob::Union{DiffEqBase.AbstractODEProblem,DiffEqBase.AbstractDAEProblem},
+                            alg::GlobalAdjoint, args...;
+                            dt,kwargs...)
+
+    opt = Dict(kwargs)
+    otheropts = delete!(copy(opt), :dt)
+    tstops = get(opt, :tstops, range(prob.tspan[1], stop=prob.tspan[2], step=dt))
+    local sol
+
+    n = length(prob.u0)
+    k = 2 # 2 random vectors
```

Comment:
would be good to generalize this to an option

Reply:
I will begin working on this as soon as I get the rest of the algorithm working. The paper mentions "two random vectors can achieve 99% probability of accuracy to a factor of 10", and most of their work is shown with k=2. Once I get a reliable algorithm, I will begin varying k and check for consistency.